src/registration/CommunicationPreferencesCheckbox.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
+import { LabeledCheckbox } from 'src/components/common';
+
+interface CommunicationPreferencesCheckboxProps {
+  title: string;
+  checked: boolean;
+  onChange?: (value: boolean) => void;
+}
+
+export const CommunicationPreferencesCheckbox = (props: CommunicationPreferencesCheckboxProps): ReactNode => {
+  const { title, checked, onChange } = props;
+  return (
+    <div style={{ marginTop: '.25rem' }}>
+      <LabeledCheckbox checked={checked} disabled={onChange === undefined} onChange={onChange}>
```

Comment:
FYI, tsx also supports literal property shorthand:
```
      <LabeledCheckbox checked onChange disabled={onChange === undefined}>
```

There's nothing wrong with how this is written at all, but I figured I'd point it out, since this is the first usage of tsx in this project.

Reply:
`<Component prop />` is equivalent to `<Component prop={true} />`, not `<Component prop={prop} />`.

Reply:
You _could_ use the spread operator in JSX to do something like `<Component {...{ prop1, prop2 }} />`. But I think that's non-idiomatic and not easy to read.

Reply:
For simple functional components, just making the `props` inputs line up with the inputs to the element would allow us to do the same splatting operator, which would be pretty concise.

Reply:
I think this is a case where, if we want to use JSX, we should lean into its idioms. Or if we decide we don't like those idioms, maybe stick with hyperscript helpers.

Reply:
FWIW, the React Docs actually call out splatting props: https://react.dev/learn/passing-props-to-a-component#forwarding-props-with-the-jsx-spread-syntax

Reply:
Yes, it's frequently used for passing through props. However, this is different in that it's creating an object and spreading it in the same line.